src/map/atcommand.cpp

```diff
@@ -5835,6 +5835,14 @@ ACMD_FUNC(useskill)
 		return -1;
 	}
 
+	if (!skill_id) {
```

Comment:
hi maybe put this also to avoid error in console :D
[Error]: Skill '65535' is undefined! skill.cpp:191::skill_check 

`if (!skill_id && skill_id < MAX_SKILL_ID) {`